### PR TITLE
[ci:docs] Fix man page for buildah push

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: docs/buildah-push.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Layer(s) to encrypt: 0-indexed layer indices with support for negative indexing
 
 **--format**, **-f**
 
-Manifest Type (oci, v2s1, or v2s2) to use when saving image to directory using the 'dir:' transport (default is manifest type of source)
+Manifest Type (oci, v2s2, or v2s1) to use when pushing an image. (Supported for **dir** and **docker** transports)
```
Comment (vrothberg) [inline docs/buildah-push.md:84]: Please drop the entire last sentence.  It also support containers-storage.  In fact it supports all transports but the ones with implicit conversion (oci, oci-archive, docker-archive).

**Comment by rhatdan** [comment]
@saschagrunert @nalind @vrothberg @giuseppe @TomSweeneyRedHat @umohnani8 PTAL
@containers/podman-maintainers PTAL

File: docs/buildah-push.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Layer(s) to encrypt: 0-indexed layer indices with support for negative indexing
 
 **--format**, **-f**
 
-Manifest Type (oci, v2s1, or v2s2) to use when saving image to directory using the 'dir:' transport (default is manifest type of source)
+Manifest Type (oci, v2s2, or v2s1) to use when pushing an image. (default is manifest type of the souce image)
```
Comment (TomSweeneyRedHat) [inline docs/buildah-push.md:84]:
```suggestion
Manifest Type (oci, v2s2, or v2s1) to use when pushing an image. (default is manifest type of the source image)
```

**Review by saschagrunert — APPROVED** [review]
LGTM

**Review by vrothberg — COMMENTED** [review]
/lgtm

**Pull request merged**